othello/moderator/utils.py

```diff
@@ -0,0 +1,98 @@
+import enum
+import importlib.machinery
+import operator
+from functools import partial, reduce, wraps
+
+from . import constants
+
+
+class UserError(enum.Enum):
+
+    NO_MOVE_ERROR = (-1, "No move submitted")
+    READ_INVALID = (-2, "Submitted move is not an integer or within range")
+    INVALID_MOVE = (-3, "{move} is invalid for {player} on {board}")
+
+
+class ServerError(enum.Enum):
+
+    TIMEOUT = (-4, "Timed out reading from subprocess")
+    UNEXPECTED = (-5, "Unexpected error")
+    PROCESS_EXITED = (-6, "Process exited unexpectedly")
+    FILE_DELETED = (-7, "Player script cannot be found on the server")
+    DISCONNECT = (-8, "Unexpectedly disconnected from socket")
+
+
+class Generator:
+    def __init__(self, gen):
+        self.gen = gen
+        self.return_value = None
+
+    def __iter__(self):
+        self.return_value = yield from self.gen
+
+
+def capture_generator_value(f):
+    @wraps(f)
+    def g(*args, **kwargs):
+        return Generator(f(*args, **kwargs))
+
+    return g
+
+
+def import_strategy(path):
+    return importlib.machinery.SourceFileLoader("strategy", path).load_module().Strategy()
+
+
+bit_or = partial(reduce, operator.__or__)
```

Comment:
You've got a bunch of functions messing around with bits here. Can you add docstrings so it's easier to tell what they're doing? The names aren't extremely descriptive and the variable names aren't either.